.editorconfig

```diff
@@ -0,0 +1,56 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = space
```

Comment:
Perhaps this will be `indent_size = 4`
According to https://editorconfig.org/

Reply:
Sorry, I did not paid attention to it. Fixed 👍